testdata/m1_facilitator/facilitator_manifest.yml

```diff
@@ -0,0 +1,28 @@
+version: v0.14
+architecture_layout: M1
+personas:
+  - facilitator
+chain: 1405
```

Comment:
We dont need this for now, its more related to M0

Reply:
Removed!